File: URI-Judge-Online-python-language/1038.py
lanche = input().split(" ")
codigo=int(lanche[0])
quantidade=int(lanche[1])

if codigo == 1:
    resultado = 4*quantidade
    print("Total: R$","%.2f"%resultado)

if codigo == 2:
    resultado = 4.50*quantidade
    print("Total: R$","%.2f"%resultado)
if codigo == 3:
    resultado = 5*quantidade
    print("Total: R$","%.2f"%resultado)
if codigo == 4:
    resultado = 2*quantidade
    print("Total: R$","%.2f"%resultado)
if codigo == 5:
    resultado = 1.50*quantidade
    print("Total: R$","%.2f"%resultado)

File: URI-Online-Judge-c/1038.c
#include <stdio.h>

int main(){

int codigo;
int qtd;
float x;
scanf("%d %d", &codigo, &qtd);
switch(codigo){
    case 1:x=qtd*4;
    printf("Total: R$ %.2f\n",x);
    break;
    case 2:x=qtd*4.50;
    printf("Total: R$ %.2f\n",x);
    break;
    case 3:x=qtd*5;
    printf("Total: R$ %.2f\n",x);
    break;
    case 4:x=qtd*2;
    printf("Total: R$ %.2f\n",x);
    break;
    case 5:x=qtd*1.5;
    printf("Total: R$ %.2f\n",x);
    break;
}
return 0;
}
